tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -228,6 +311,62 @@ exports[`hug.js 1`] = `
 
 `;
 
+exports[`logical-expression.js 1`] = `
+<div>
+  {a || "b"}
+</div>;
+
+<div>
+  {a && "b"}
+</div>;
+
+<div>
+  {a || <span></span>}
+</div>;
+
+<div>
+  {a && <span></span>}
+</div>;
+
+<div>
+  {a && <span>
```

Comment:
You commented on the unformatted code, so I'm a bit confused.

Current behavior in this PR is to always wrap parens (and add line breaks) around JSX in `&&`/`||`, whether it's breaking or not.